### PR TITLE
README: Convert HTML to legit MarkDown

### PR DESCRIPTION
**Pull request by Lord-Haji**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-Zarel - The main workspace we used to begin Exiled off of.<br>
-<b>Retired Developers</b><br>
-Sukesha - Former co-owner, HTML developments, and CSS creator.<br>
-Alpha Hawk - Former Developer.<br>
-<b>Thanks for reading!  Hope you enjoy these codes, that are possible thanks to the above listed.</b>
+### To start:
+Use ``nvm install 6.6.0``, after Node 6.6.0 is installed, then run ``node app.js`` to start the server locally at **Config.port**
+After you install version 6.6.0, every other time you will have to do "nvm use 6.6.0", then "node app.js" as usual to startup your server.
+### Developers:
+Insist - Main developer of Exiled files, Sysop.
```
Comment (DeathlyPlays) [inline README.md]: These didn't go to next line

**Review by DeathlyPlays — CHANGES_REQUESTED** [review]
Definitely need the different lines

**Review by ghost — CHANGES_REQUESTED** [review]
I saw the finished product, it doesn't look pretty.

**Comment by DeathlyPlays** [comment]
A lot of stuff needs to be changed there's tons of hideous things without using the br...

File: README.md [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
-<b>Retired Developers</b><br>
-Sukesha - Former co-owner, HTML developments, and CSS creator.<br>
-Alpha Hawk - Former Developer.<br>
-<b>Thanks for reading!  Hope you enjoy these codes, that are possible thanks to the above listed.</b>
+### To start:
+Use ``nvm install 6.6.0``, after Node 6.6.0 is installed, then run ``node app.js`` to start the server locally at **Config.port**
+After you install version 6.6.0, every other time you will have to do "nvm use 6.6.0", then "node app.js" as usual to startup your server.
+### Developers:
+Insist - Main developer of Exiled files, Sysop.\n
+Volco - Fixes errors, implements features, Sysop.\n
```
Comment (DeathlyPlays) [inline README.md]: All these \n's mean nothing

**Review by DeathlyPlays — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
-Back At My Day - Developments.<br>
-Mystifi - Assistance, Developments.<br>
-AlfaStorm & Ashley The Pikachu - Minor developments.<br>
-Zarel - The main workspace we used to begin Exiled off of.<br>
-<b>Retired Developers</b><br>
-Sukesha - Former co-owner, HTML developments, and CSS creator.<br>
-Alpha Hawk - Former Developer.<br>
-<b>Thanks for reading!  Hope you enjoy these codes, that are possible thanks to the above listed.</b>
+### To start:
+Use ``nvm install 6.6.0``, after Node 6.6.0 is installed, then run ``node app.js`` to start the server locally at **Config.port**
```
Comment (DeathlyPlays) [inline README.md]: Now just neaten up this portion

**Review by DeathlyPlays — CHANGES_REQUESTED** [review]

**Review by DeathlyPlays — APPROVED** [review]

**Pull request merged**